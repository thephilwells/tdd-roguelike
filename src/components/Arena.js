import React from 'react'
import { render, Box, Text } from 'ink'

export const Arena = () => (
  <Box display="flex" padding={2} margin={1} align-self="flex-start" borderStyle="double">
    <Text>foo</Text>
  </Box>
)
