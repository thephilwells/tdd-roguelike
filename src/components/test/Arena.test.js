import React from 'react'
import { render } from 'ink-testing-library'
import { Arena } from '../Arena'

describe('ARENA', () => {
  it('returns an empty arena container box', () => {
    const { lastFrame } = render(<Arena />)
    
    expect(lastFrame()).toMatchSnapshot()
  })
})
