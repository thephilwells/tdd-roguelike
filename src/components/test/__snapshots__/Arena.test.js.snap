// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ARENA returns an empty arena container box 1`] = `
"






































"
`;
